### DISCUSSION TITLE
Thank you for starting this!

### DISCUSSION THREAD

**Discussion by Rot-gut** [Dickdebonair/Brightis-fan-translation#19] [General]
I picked up a sealed copy of Brightis in Taiwan a few months back but was still yearning for an English fan patch. I'm not sure I can provide any help but just wanted to share my enthusiasm and best wishes on this project.

**Comment by Dickdebonair** [comment]
Glad to see your support! 
We may open up a testing version once the text has been properly patched, so stay tuned!
